systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaBridgeResource.java

```diff
@@ -93,7 +92,7 @@ private static KafkaBridge waitFor(KafkaBridge kafkaBridge) {
         String kafkaBridgeCrName = kafkaBridge.getMetadata().getName();
 
         LOGGER.info("Waiting for Kafka Bridge {}", kafkaBridgeCrName);
-        DeploymentUtils.waitForDeploymentReady(KafkaBridgeResources.deploymentName(kafkaBridgeCrName), kafkaBridge.getSpec().getReplicas());
+        KafkaBridgeUtils.waitForKafkaBridgeReady(kafkaBridgeCrName);
```

Comment:
Should this pass the replicas as well to check the pods as it was doing before?

Reply:
After discussion with @Frawless and @see-quick we agreed that we don't really need to know if the pods are in ready status, because if they're not, the CR or deployment itself is not in ready state.

Please correct me if I'm wrong @scholzj ..